examples/fluids/navierstokes.c

```diff
@@ -107,19 +107,25 @@ int main(int argc, char **argv) {
   CeedInit(app_ctx->ceed_resource, &ceed);
   user->ceed = ceed;
 
+  {  // Set error handler
+    PetscErrorCode ierr;
+    ierr = CeedSetErrorHandler(ceed, CeedErrorStore);
+    if (ierr != CEED_ERROR_SUCCESS) SETERRQ(comm, PETSC_ERR_LIB, "Setting libCEED error handler failed");
+  }
```

Comment:
I updated this in Ratel to this approach so we don't need to have an `ierr`
```suggestion
  PetscCheck(CeedSetErrorHandler(ceed, CeedErrorStore) == CEED_ERROR_SUCCESS, comm, PETSC_ERR_LIB, "Setting libCEED error handler failed");
```

Reply:
Side question - PETSc uses `PETSC_SUCCESS` instead of `PETSC_ERROR_SUCCESS`. I've wondered if we should do the same (totally not an important question)

Reply:
I see both sides. `CEED_ERROR_SUCCESS` keeps the enum prefix consistent for all libCEED return codes. But also having "error" and "success" in the same name is a bit jarring.

I'd lean towards `CEED_SUCCESS`, but don't have a strong preference.